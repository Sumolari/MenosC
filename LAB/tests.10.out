Linea 4: syntax error, unexpected INC_, expecting SQBROP_ or ASSIGN_
