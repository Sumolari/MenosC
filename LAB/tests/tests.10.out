Linea 3: Tipo incompatible
Linea 4: syntax error, unexpected INC_, expecting SQBROP_ or ASSIGN_

Numero de errores:      2
